cluster-autoscaler/provisioningrequest/provreqwrapper/wrapper_test.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provreqwrapper
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1"
+)
+
+func TestProvisioningRequestWrapper(t *testing.T) {
```

Comment:
Would it make sense to make this a table-based test? There are only 2 cases (all templates and missing templates), but this is a recommended test writing pattern.

Reply:
Not sure it worth in current test, since it test the whole wrapper and here we have only one wrapper. In case we would like to extend the test to multiple wrappers it would make sense, but currently making the test table-based complicate the test.